src/InfoViz/Native/HistogramSelector/index.js

```diff
@@ -843,6 +819,8 @@ const DEFAULT_VALUES = {
   defaultScore: 0,
   dragMargin: 8,
   selectedDef: null,
+
+  numberOfBins: 32,
```

Comment:
I don't feel adding a provider for 3 lines of code. That should be managed within our glue code not in the generic one.